py-rse/py-scripting.Rmd

```diff
@@ -733,27 +905,6 @@ df.plot.scatter(x='word_frequency', y='inverse_rank',
 knitr::include_graphics("figures/py-rse-scripting/jane_eyre.png")
 ```
 
-Using `script_template.py` as a guide,
-take this plotting code and write a new python script called `plotcounts.py`.
-The script should:
-
-a. Use the `type=argparse.FileType('r')`, `nargs='?'` and `default='-'` options
-   for the input file argument (i.e. similar to the `countwords.py` script)
-   so that `plotcounts.py` uses standard input if no csv file is given.
-
-b. Include an optional `--outfile` argument for the name of the output image file.
-   The default value should be `plotcounts.png`.
-
-b. Include an optional `--xlim` argument so that the user can change the x-axis bounds.
-
-Once you've written `plotcounts.py`,
-generate a plot of the word counts for *Jane Eyre*:
-
-```shell
-$ python plotcounts.py ../results/jane_eyre.csv --outfile ../results/jane_eyre.png
-```
-
-
 ## Summary {#py-rse-py-scripting-summary}
```

Comment:
I think this summary section can be removed. It's kind of redundant (because of the key points section) and this particular summary statement attempts to link the contents of this chapter to the novice book. Since the RSE book is being published first, we can't be referred back to a novice book that doesn't exist.

Reply:
Relocated a paragraph to the end.